### ISSUE TITLE
Arrow button accessibility issue

### ISSUE THREAD

**Issue by cBridges851** [cBridges851/tkinter-image-viewer#5]
**Describe the bug**
Pressing enter or space on the arrow buttons don't move onto the next or previous image.

This might be problematic because some users are unable to use a mouse due to physical barriers, like motor impairments. Read more about accessibility here: https://www.w3.org/WAI/standards-guidelines/wcag/

**To Reproduce**
1. Open the application
2. Tab to one of the arrow buttons
3. Press enter or space
4. The image doesn't change

**Expected behavior**
If the left arrow button is in focus and the enter or space button is clicked, then the image should change to the previous image if it's not the first.

If the right arrow button is in focus on the enter or space button is clicked then the image should change to the next image if it's not the last.

**Screenshots**
![image](https://user-images.githubusercontent.com/54030147/193460687-ce7fb66c-6797-4587-a12a-31edde30a2d2.png)
These are the buttons I'm referring to.

**Desktop (please complete the following information):**
 - OS: Windows 10

**Additional context**
This was not originally thought of when I created this application. You can use the left or right keys on the keyboard, but the user isn't made aware of this.

This issue is open to being worked on for Hacktoberfest, comment below if you have any questions and/or would like to work on it! First come first serve.

**Comment by cBridges851** [comment]
Hi @i-m-karanagarwal! You are more than welcome to work on this issue, I'll assign you to this now 😊!

Please let me know if you have any questions while you work on this!

**Comment by cBridges851** [comment]
Hi @i-m-karanagarwal, the enter and the space button should do the same thing, they should cause the next or previous image to display in the viewer when the user presses the key.

E.g. Imagine we have image1.png, image2.png, and image3.png in the images folder, and the image viewer displays image1.png when you open the image viewer.

When you tab to the right button and press enter or space, it should move onto image 2.png.

If you then tab to the left button and press enter or space, it should go back to image1.png.


I hope that helps?
